ts/torch_handler/request_envelope/kservev2.py

```diff
@@ -29,6 +29,7 @@
 
 # NOTE: numpy has more types than v2 protocol
 _NumpyToDatatype["object"] = "BYTES"
+_NumpyToDatatype["U"] = "BYTES"
```

Comment:
What does “U” mean for numpy?

Reply:
U stands for "Unicode String" - https://www.w3schools.com/python/numpy/numpy_data_types.asp

Reply:
good to leave a comment here with the reference?

Reply:
why not !! Added the comment